playbooks/roles/katello_provisioning/tasks/katello.yml

```diff
@@ -118,13 +118,6 @@
     --unlimited-hosts
   when: katello_provisioning_activation_key.stderr.find('not found') != -1
 
-- name: 'medium env should be in locs'
```

Comment:
@stbenjam should be able to judge this change best

Could you provide an idea behind what this change is needed for?